packages/builder/src/stores/builder/screens.ts

```diff
@@ -13,15 +13,33 @@ import {
 import { createHistoryStore } from "@/stores/builder/history"
 import { API } from "@/api"
 import { BudiStore } from "../BudiStore"
+import {
+  FetchAppPackageResponse,
+  DeleteScreenResponse,
+  Screen,
+  Component,
+  SaveScreenResponse,
+} from "@budibase/types"
+import { ComponentDefinition } from "./components"
+
+interface ScreenState {
+  screens: Screen[]
+  selectedScreenId?: string
+  selected?: Screen
```

Comment:
I can't see any external usages of this `selected` property other than in the `afterAction` of the history store constructor, so it'll always be null... I'm assuming this is maybe a leftover and we can refactor it out? I think in after action we should be reading the `selectedScreen` derived store and using that to select a component instead of this `selected` property, which we can remove.